<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-0HRE68P" failures="1" tests="3" name="Test" time="26.422" errors="0" timestamp="2024-05-09T07:52:24 IST">
  <testcase classname="Facebook.Testcase3" name="tc3" time="2.215"/>
  <testcase classname="Facebook.testcase2" name="tc2" time="1.879"/>
  <testcase classname="Facebook.Testcase1" name="tc1" time="6.095">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [456tghthjkhrtj] but found [Error]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [456tghthjkhrtj] but found [Error]
at Facebook.Testcase1.tc1(Testcase1.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tc1 -->
</testsuite> <!-- Test -->
